From 8f83e8f7ad86e6dbd1da8512ad41bfb1034d2c0e Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Mon, 12 Mar 2018 19:47:24 +0100
Subject: [PATCH 1/1] efi_selftest: fix device tree unit test

Include libfdt.h was moved by commit b08c8c487083 ("libfdt: move headers to
<linux/libfdt.h> and <linux/libfdt_env.h>")

Fixes: e236200c7fa6 ("efi_selftest: check installation of the
       device tree")
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_selftest/efi_selftest_fdt.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/efi_selftest/efi_selftest_fdt.c b/lib/efi_selftest/efi_selftest_fdt.c
index 24db0dcf7d5..e5a8d6a6ae5 100644
--- a/lib/efi_selftest/efi_selftest_fdt.c
+++ b/lib/efi_selftest/efi_selftest_fdt.c
@@ -12,7 +12,7 @@
  */
 
 #include <efi_selftest.h>
-#include <libfdt.h>
+#include <linux/libfdt.h>
 
 static struct efi_boot_services *boottime;
 static const char *fdt;
-- 
2.16.1

